### ISSUE TITLE
Ignore skill updates which don't trigger a quest requirement

### ISSUE THREAD

**Issue by BreakBB** [Questie/Questie#6057]
Currently `QuestieProfessions:Update` triggers `AvailableQuests.CalculateAndDrawAll` on every skill update. We can improve this, because only a few skill break points are actually relevant for quests (25, 50, 75, ...).

**Comment by Cabro** [comment]
Keep in mind there are some quests/crafts that give you 5x skill points, and if you check specifically for 25, 50, 75, they might never trigger.

**Comment by BreakBB** [comment]
Yeah, we need to make sure to change this check for available quests, not objectives. We might also be able to ignore some professions entirely.
